### DISCUSSION TITLE
What was the name of the 🚀 in the original 👽?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1102] [Galaxy Trivia] [answered]
**Bonus:** Who is credited with designing the first concept 🚁?

**Comment by peterbaccaro** [comment]
The spacecraft in the original "Alien" film (1979) is called the **Nostromo**.

**Paul Cornu**, a French bicycle maker, is credited with inventing the helicopter in 1907. His invention is the first helicopter as we know it today, and it managed to lift off the ground about 1 foot, for about 20 seconds.

**Igor Sikorsky**, a Russian-American aviation pioneer, designed and built the first successful single-rotor helicopter, known as the VS-300, which made its first successful free flight in 1939.

**Comment by thuongtruong1009** [comment]
The spaceship in the original 1979 film "Alien" was called the "Nostromo." It was a commercial towing vessel.

The first concept helicopter is attributed to Igor Sikorsky, a Russian-American aviation pioneer. Sikorsky designed and built the first successful helicopter, the VS-300, which made its first flight in 1939. This achievement marked a significant milestone in the development of modern helicopters.

**Comment by sheri-na** [comment]
The spaceship is named The United States Cargo Star Ship (USCSS) Nostromo and the person who was credited for the first concept of a helicopter is Igor Sikorsky.

**Comment by ParanoidUser** [comment]
**Extra 1:** Where did the crew first find the 👽🥚 on the planet?
**Extra 2:** What 🔫 was used to fight off the 👽 in the climactic scene?

**Comment by sheri-na** [comment, marked as answer]
The crew first find for eggs in Alien (1979) was on the Derelict spacecraft within the ship's cargo hull on LV-426 moon, it was the crew first encounter with the face hugger.

In the last scene when Ripley confronts the alien in the escape shuttle, she is using a harpoon gun.
